package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "description": "GitHub Apps toolset for Node.js",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/octokit/app.js.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
we do all the other packages under MIT so I’d do the same here

Reply:
I just hit enter through all the package.json creation fields, but happy to change that :)